mobile/native/src/db/models.rs

```diff
@@ -83,6 +83,16 @@ impl Order {
         }
     }
 
+    /// Sets all filling orders to failed. Only be used for emergency recoveries!
+    pub fn set_all_filling_orders_to_failed(conn: &mut SqliteConnection) -> Result<()> {
+        diesel::update(orders::table)
+            .filter(schema::orders::state.eq(OrderState::Filling))
+            .set(orders::state.eq(OrderState::Failed))
+            .execute(conn)?;
```

Comment:
Do we get a `updated_rows` count here? If so, it might be worth logging this. This could help us debug things in the future.